### PR TITLE
[ci skip]Update change log and authors

### PR DESCRIPTION
**Pull request by xiaofeng11**
@pandamicro Please review it, Thanks.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,30 +1,74 @@
...
+
+[NEW]           Core: Added Application::getVersion() to get the app version.
 [NEW]           UI: Add PageView indicator.
-[NEW]           Label: Add three Overflow type to new label, see release note for more information.
+[NEW]           UI: Label add three Overflow type to new label, see release note for more information.
 [NEW]           UI: UIText::clone supports clone the text effect.
 [NEW]           Label: Add methods to query label effect state.
+[NEW]           UI: UIRichText support the new line element.
+[NEW]           UI: Scale9Sprite fade actions with cascade opacity.
+[NEW]           UI: Pageview support adjust child size.
```
Comment (zilongshanren) [inline CHANGELOG]: This one `+[NEW]           UI: Pageview support adjust child size.` is not needed.
It's a part of `[HIGHLIGHT]     UI: Change PageView to derive from ListView.`

Comment (zilongshanren) [inline CHANGELOG]: `[NEW]           UI: Scale9Sprite fade actions with cascade opacity.` This one is also not needed.
I rewrite the Scale9Sprite and it's an issue of my code. It's not an issue of the previous version.

File: CHANGELOG [hunk 2]
```diff
@@ -1,30 +1,74 @@
...
 [FIX]           Audio: Fix `FinishCallback` never be called in Windows.
-[FIX]           UI:  Fix Layout stencil clipping nested with Clipping Node rendering issue.
-[FIX]           UI:  Keyboard doesn't hide when click the screen outside of EditBox on iOS platform.
+[FIX]           UI: Fix Layout stencil clipping nested with Clipping Node rendering issue.
+[FIX]           UI: Keyboard doesn't hide when click the screen outside of EditBox on iOS platform.
+[FIX]           UI: Fix a fatal bug in EditBox implement on Windows platform.
+[FIX]           UI: Fix edit box setPlaceholderFontName and scale font size issue.
+[FIX]           Core: Fix memory leak when initWithImage() failed.
+[FIX]           Network: CCDownloader on iOS is broken in v3.9 js-binding.
+[FIX]           UI: Fix label clamp when scaled with bmfont.
```
Comment (zilongshanren) [inline CHANGELOG]: `[FIX]           UI: Fix label clamp when scaled with bmfont.` This one is not needed.

Comment (zilongshanren) [inline CHANGELOG]: `[FIX]           UI: Fix label clamp when scaled with bmfont.` This one is not needed.

File: CHANGELOG [hunk 3]
```diff
@@ -1,30 +1,74 @@
...
+[REFINE]        Network: Crash when removing a remotely downloaded image from texture cache in js-binding.
+[REFINE]        Win10: WinRT project update version to v3.10.
+[REFINE]        Console: Add quiet option for Cocos Toolkit.
+[REFINE]        JS: New GC model for js-binding.
+[REFINE]        Doc: Fix typos in documentation and comments.
+[REFINE]        UI: update controlButton size calculate with new Scale9Sprite logic.
+[REFINE]        Win10: Added missing _USRJSSTATIC preprocessor define for ARM builds.
+[REFINE]        JS: Added ccvector_to / ccmap_to converted to new js-binding API.
+[REFINE]        UI: Slider misprint fix.
+[REFINE]        UI: Scale9Sprite cropped sprite frames support.
```
Comment (zilongshanren) [inline CHANGELOG]: `[REFINE]        UI: Scale9Sprite cropped sprite frames support.` This one is not needed. The previous version of Scale9Sprite support this.

**Comment by xiaofeng11** [comment]
@zilongshanren Remove unnecessary log as comment, please review again.

**Comment by zilongshanren** [comment]
@xiaofeng11  I'm OK with this PR.

**Comment by zilongshanren** [comment]
@xiaofeng11 
Next time, please add [ci skip] to the title of this PR, thanks.

**Pull request merged**